apps/tailwind-components/components/form/FieldInput.vue

```diff
@@ -0,0 +1,129 @@
+<script setup lang="ts">
```

Comment:
we dropped this file right and instead use Input.vue?

Reply:
Yes, I thought we did as well. I'll update this branch with the master